bok_choy/a11y/axe_core_ruleset.py

```diff
@@ -195,7 +195,7 @@ def customize_ruleset(self, custom_ruleset_file=None):
         if not custom_file:
             return
 
-        with open(custom_file) as additional_rules:
+        with open(custom_file, encoding="utf8") as additional_rules:
```

Comment:
pylint version [2.10.0](https://github.com/PyCQA/pylint/releases/tag/v2.10.0) deprecated the use of open() without explicitly specifying any encoding. See https://github.com/PyCQA/pylint/issues/3826 for details.